bot/exts/advent_of_code/_cog.py

```diff
@@ -128,7 +130,7 @@ async def completionist_task(self) -> None:
                 await members.handle_role_change(member, member.add_roles, completionist_role)
 
     @commands.group(name="adventofcode", aliases=("aoc",))
-    @whitelist_override(channels=AOC_WHITELIST)
+    @in_whitelist(channels=AOC_WHITELIST, redirect=AOC_REDIRECT)
```

Comment:
I think this change is the cause of the below behaviour:

Previously the admin commands block and refresh could be ran anywhere, so long as you're an admin, but now the bot forces you to run them in this whitelist, which doesn't seem desirable for block especially